app/views/shared/user/_sidebar.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="col-md-2 container-height shadow-box sticky-top">
+  <div class="container-md">
+    <div class="row p-3">
+      <div class="btn-group btn-entity-group">
+        <%= button_to 'Project', projects_development_metrics_path, class: 'btn btn-secondary entity-btn', method: :get %>
```

Comment:
I see that this sidebar with these three buttons is repeated 3 times. Perhaps we reuse the same partial, and use a conditional to decide which select_tag to use. Not a blocker for this PR though

Reply:
roger that!